---
header:
  image: test2.jpg
permalink: /
excerpt: "Fritz Sedlazeck main homepage."
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

My research focuses on the development of scalable algorithms to assess the variability in genomes in terms of single nucleotide polymorphisms (SNP) as well as structural variations (SVs) within and among populations. I have a strong background in computer science, especially in leveraging high throughput architectures ranging from highly parallel graphics cards, over clusters, or cloud architectures. My training in genomics started in 2008, when I started my PhD in computational biology where I designed algorithms to investigate genomic variability of model and non-model organisms through the alignment of short reads. During my PhD and following my entire career I was always intrigued to investigate genomic variability being it in across tissues as well as within populations or across populations. This drove my training and the fascination how variable genomes are and investigating the impact of such variability. To assess these, I collaborated and initiated efforts to determine which technologies or methodologies were most appropriate to discover variations of different forms as well as the biases that they contain. This work just recently leads to novel insights in structural variations and their genotypic and phenotypic impact. In 2014 I joined the lab of Michael Schatz at Cold Spring Harbor Laboratory and later at Johns Hopkins University to broaden my knowledge about de novo assemblies and their possibilities to assess variability. Over these two years I obtained insights into novel technologies (long reads and linked reads), which often allow a more precise detection of genomic variability and its assessment with respect to the individual haplotypes.

Over my young career as a scientist I followed a clear path to investigate genomic variability starting with SNPs and focusing over the last 5 years on structural variations. I have successfully co supervised 5 students, initiated successful international collaborations as well as lead three projects resulting in last author publications.

# Recruiting
If you want to join a young but aspiring lab please check out [Apply](apply/){: .btn .btn--success}.   
I welcome applications for a Computational Biology PostDoc, who is motivated and interested in the dynamics of genomes to improve our understanding of SVs. We benefit from a thrilling environment here at the Human Genome Sequencing Center at Baylor College of Medicine.


# Recent News

+ Preprint:  [xAtlas: Scalable small variant calling across heterogeneous next-generation sequencing experiments](https://www.biorxiv.org/content/early/2018/04/05/295071)

+ Preprint:  [Detection of GBA missense mutations and other variants using the Oxford Nanopore MinION](https://www.biorxiv.org/content/early/2018/04/03/288068)

+ Published: [Piercing the dark matter: bioinformatics of long-range sequencing and mapping](https://www.nature.com/articles/s41576-018-0003-4?WT.feed_name=subjects_biological-sciences)

+ Published: [LRSim: a Linked Reads Simulator generating insights for better genome partitioning](http://www.sciencedirect.com/science/article/pii/S2001037017300855)

