---
header:
  image: test2.jpg
title: "Advanced Methylome Analysis after Bisulfite Deep Sequencing: an Example in Arabidopsis"
collection: publications
permalink: /publication/2012-07-20-Advanced Methylome Analysis after Bisulfite Deep Sequencing an Example in Arabidopsis-number-3
excerpt: ''
tags:
  - Methylation
  - Plant genomics
  - Arabidopsis thaliana
  - Sequence alignment 
date: 2012-07-20
venue: 'PLoS ONE'
paperurl: 'http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0041528'
citation: 'H.Q. Dinh, M. Dubin*, <b>F.J. Sedlazeck*</b>, N. Lettner, O. Mittelsten Scheid, and A. von Haeseler (2012). &quot;Advanced Methylome Analysis after Bisulfite Deep Sequencing: an Example in Arabidopsis.&quot; <i>Genome Research</i>. 22, 1468-1476.'
---

Deep sequencing after bisulfite conversion (BS-Seq) is the method of choice to generate whole genome maps of cytosine methylation at single base-pair resolution. Its application to genomic DNA of Arabidopsis flower bud tissue resulted in the first complete methylome, determining a methylation rate of 6.7% in this tissue. BS-Seq reads were mapped onto an in silico converted reference genome, applying the so-called 3-letter genome method. Here, we present BiSS (Bisufite Sequencing Scorer), a new method applying Smith-Waterman alignment to map bisulfite-converted reads to a reference genome. In addition, we introduce a comprehensive adaptive error estimate that accounts for sequencing errors, erroneous bisulfite conversion and also wrongly mapped reads. The re-analysis of the Arabidopsis methylome data with BiSS mapped substantially more reads to the genome. As a result, it determines the methylation status of an extra 10% of cytosines and estimates the methylation rate to be 7.7%. We validated the results by individual traditional bisulfite sequencing for selected genomic regions. In addition to predicting the methylation status of each cytosine, BiSS also provides an estimate of the methylation degree at each genomic site. Thus, BiSS explores BS-Seq data more extensively and provides more information for downstream analysis.

[Download paper here](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0041528)

