---
header:
  image: test2.jpg
title: "Teaser: Individualized benchmarking and optimization of read mapping results for NGS data"
collection: publications
permalink: /publication/2015-10-22-Teaser Individualized benchmarking and optimization of read mapping results for NGS data-number-11
excerpt: ''
tags:
  - NGS methods
  - Sequence alignment
  - Benchmark
date: 2015-10-22
venue: 'Genome Biology'
paperurl: 'https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0803-1'
citation: 'M. Smolka, P. Rescheneder, M.C. Schatz, A. von Haeseler, <b>F.J. Sedlazeck</b> (2015). &quot;Teaser: Individualized benchmarking and optimization of read mapping results for NGS data.&quot; <i>Genome Biology</i>. 16, 235.'
---

Mapping reads to a genome remains challenging, especially for non-model organisms with lower quality assemblies, or for organisms with higher mutation rates. While most research has focused on speeding up the mapping process, little attention has been paid to optimize the choice of mapper and parameters for a userâ€™s dataset. Here, we present Teaser, a software that assists in these choices through rapid automated benchmarking of different mappers and parameter settings for individualized data. Within minutes, Teaser completes a quantitative evaluation of an ensemble of mapping algorithms and parameters. We use Teaser to demonstrate how Bowtie2 can be optimized for different data.

[Web server](http://teaser.cibiv.univie.ac.at/)
[Github](https://github.com/cibiv/teaser/)

[Download paper here](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0803-1)

