---
header:
  image: test2.jpg
title: "Ectodysplasin signalling genes and phenotypic evolution in sculpins (Cottus)"
collection: publications
permalink: /publication/2015-09-09-Ectodysplasin signalling genes and phenotypic evolution in sculpins-number-9
excerpt: ''
tags:
  - Eda
  - Evolution
  - Fish genomics
  - Sequence alignment
date: 2015-09-09
venue: 'Proceedings B'
paperurl: 'http://genome.cshlp.org/content/22/8/1468.abstract'
citation: 'J. Cheng, <b>F.J. Sedlazeck</b>, J. AltmuÌˆller, A.W. Nolte (2015). &quot;Ectodysplasin signalling genes and phenotypic evolution in sculpins (Cottus).&quot; <i>Proceedings B</i>. 282, 20150746.'
---

Despite their deeply conserved function among vertebrates, ectodysplasin (Eda) signalling genes are involved in microevolutionary change in humans and sticklebacks. If such a dual role is common, Eda signalling genes constitute hotspots for morphological evolution. Variation in sculpin (Cottus) skin prickling and body shape resembles patterns caused by variation in Eda signalling in sticklebacks. We mapped Eda signalling genes and performed quantitative trait locus mapping in crosses between Cottus rhenanus and Cottus perifretum. A genomic region containing the Eda receptor (Edar) was strongly associated with prickling and contributed to shape. The expression of Edar in developing prickles and skeletal elements in Cottus was confirmed by in situ hybridization. Coding sequence changes between Edar alleles in C. rhenanus and C. perifretum exceeded sequence differentiation in other vertebrates. However, it is likely that additional genetic elements besides coding changes affect the phenotypic variation. Although the phenotype in a natural hybrid lineage between C. rhenanus and C. perifretum resembles C. perifretum, the respective coding Edar alleles are not fully fixed (88.6%). Hence, our results support an involvement of Eda signalling in microevolutionary changes, but imply that the Edar gene is affected by multiple evolutionary processes that vary among freshwater sculpins.

[Download paper here](http://genome.cshlp.org/content/22/8/1468.abstract)

