---
header:
  image: test2.jpg
title: "Phased diploid genome assembly with single-molecule real-time sequencing"
collection: publications
permalink: /publication/2016-10-17-Phased diploid genome assembly with single-molecule real-time sequencing-number-12
excerpt: ''
tags:
  - NGS methods
  - De novo assembly
  - SVs
  - phasing
date: 2016-10-17
venue: 'Nature Methods'
paperurl: 'https://www.nature.com/nmeth/journal/v13/n12/full/nmeth.4035.html'
citation: 'C.S. Chin, P. Peluso, <b>F.J. Sedlazeck</b>, M. Nattestad, G.T. Con- cepcion, A. Clum, C. Dunn, R.O. Malley, R. Figueroa-Balderas, A. Morales-Cruz, G.R. Cramer, M. Delledonne, C. Luo, J.R. Ecker, D. Cantu, D.R. Rank, M.C. Schatz (2016) &quot;Phased diploid genome assembly with single-molecule real-time sequencing.&quot; <i>Nature Methods</i>. 22, 1468-1476.'
---

While genome assembly projects have been successful in many haploid and inbred species, the assembly of noninbred or rearranged heterozygous genomes remains a major challenge. To address this challenge, we introduce the open-source FALCON and FALCON-Unzip algorithms (https://github.com/PacificBiosciences/FALCON/) to assemble long-read sequencing data into highly accurate, contiguous, and correctly phased diploid genomes. We generate new reference sequences for heterozygous samples including an F1 hybrid of Arabidopsis thaliana, the widely cultivated Vitis vinifera cv. Cabernet Sauvignon, and the coral fungus Clavicorona pyxidata, samples that have challenged short-read assembly approaches. The FALCON-based assemblies are substantially more contiguous and complete than alternate short- or long-read approaches. The phased diploid assembly enabled the study of haplotype structure and heterozygosities between homologous chromosomes, including the identification of widespread heterozygous structural variation within coding sequences.

[Download paper here](https://www.nature.com/nmeth/journal/v13/n12/full/nmeth.4035.html)

